.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+api-definition:
+- changed-files:
+  - any-glob-to-any-file: ['yaml/*']
```

Comment:
Does it make sense to include the pom in this section as well? Or maybe in the maintenance?

Reply:
That's a good idea. Something like `code-gen` for the label and `Code generation` for the notes section?

Maven dependency updates would also be labelled with this, as they modify the pom file. but I think they'd be excluded anyway because labelled as `dependencies`. I'll confirm

Reply:
That seems to work as expected